luigi/contrib/rdbms.py

```diff
@@ -178,6 +182,10 @@ def table(self):
     def query(self):
         return None
 
+    @abc.abstractproperty
```

Comment:
I don't think this needs to be an `abstractproperty`. You shouldn't require the user to override it. Just a plain `@property` should suffice.

Reply:
Oops, I fixed this now. Thanks.